kube/geoapi.yaml

```diff
@@ -351,11 +331,26 @@ spec:
               secretKeyRef:
                 name: rabbitmq
                 key: password
+          - name: DB_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: postgres-vm
+                key: user
           - name: DB_PASSWD
             valueFrom:
               secretKeyRef:
-                name: postgres
+                name: postgres-vm
```

Comment:
@taoteg. fyi.   in the services in kube that access the database, we have secrets/env-variables for database access.  the previous secrets were stored under the name `postgres` in kube but @sophia-massie  and I save the new ones under `postgres-vm` to not override the old ones.  so we keep temporarily the old ones that work for the kube db and then also have the new ones that work for the new db vm.